.github/workflows/fast-revert.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  pull_request_target:
+    types: [labeled]
+  workflow_dispatch:
+    inputs:
+      pr:
+        required: true
+        description: pr number
+      co_authored_by:
+        required: true
+        description: '`name <email>` for triggering user'
+
+# disable all permissions -- we use the PAT's permissions instead
+permissions: {}
+
+jobs:
+  revert:
+    runs-on: ubuntu-latest
+    if: |
+      github.event_name == 'workflow_dispatch' || github.event.label.name == 'Trigger: Revert'
+    steps:
+    - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8  # v3
+      with:
+        token: ${{ secrets.BUMP_SENTRY_TOKEN }}
+    - uses: getsentry/action-fast-revert@dc3703df06fd2774abdef6c45bfb3720a1278130  # v2.0.0
+      with:
+        pr: ${{ github.event.number || github.event.inputs.pr }}
+        co_authored_by: ${{ github.event.inputs.co_authored_by || format('{0} <{1}+{0}@users.noreply.github.com>', github.event.sender.login, github.event.sender.id) }}
+        committer_name: getsentry-bot
+        committer_email: bot@sentry.io
+        token: ${{ secrets.BUMP_SENTRY_TOKEN }}
+    - name: comment on failure
+      run: |
+        curl \
+            --silent \
+            -X POST \
+            -H 'Authorization: token ${{ secrets.BUMP_SENTRY_TOKEN }}' \
```

Comment:
you will need this secret configured

Reply:
Thanks for pointing it out. How do I go about configuring the secret? Do we use a shared secret across all sentry repositories? Or should I create one just for the snuba repository using https://github.com/Azure/actions-workflow-samples/blob/master/assets/create-secrets-for-GitHub-workflows.md

Reply:
hmmm this is a good question -- `sentry` and `getsentry` use a PAT for a bot -- I wonder if the sentry internal app (globally available) has enough permissions to write a revert

if you want to try that out you could use:

```yaml
      - uses: getsentry/action-github-app-token@38a3ce582e170ddfe8789f509597c6944f2292a9  # v1
        id: token
        with:
          app_id: ${{ secrets.SENTRY_INTERNAL_APP_ID }}
          private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
```

and then use `${{ steps.token.outputs.token }}` in place of `${{ secrets.BUMP_SENTRY_TOKEN }}`

Reply:
Did what you suggested. Mind reviewing it again?

Reply:
You are right. Removed all usages of `BUMP_SENTRY_TOKEN`